plugins/all/all.go

```diff
@@ -19,4 +19,5 @@ import (
 	_ "github.com/influxdb/telegraf/plugins/redis"
 	_ "github.com/influxdb/telegraf/plugins/rethinkdb"
 	_ "github.com/influxdb/telegraf/plugins/system"
+	_ "github.com/KPACHbIuLLIAnO4/telegraf/plugins/apache"
```

Comment:
Thank you for advice.

But I think I've not described the problem properly.

The problem is, when I clone https://github.com/KPACHbIuLLIAnO4/telegraf.git
with apache plugin and do "make build",  I get the same executable
"telegraf" as when I do "make build" in "
https://github.com/influxdb/telegraf" working copy, with the same size, and
when I do "./telegraf -usage apache", I get the message "2015/08/29
07:10:27 Plugin apache not found"




2015-08-29 4:33 GMT+03:00 Cameron Sparr <notifications@github.com>:

> In plugins/all/all.go
> <https://github.com/influxdb/telegraf/pull/153#discussion_r38257452>:
>
> > @@ -19,4 +19,5 @@ import (
> >  	_ "github.com/influxdb/telegraf/plugins/redis"
> >  	_ "github.com/influxdb/telegraf/plugins/rethinkdb"
> >  	_ "github.com/influxdb/telegraf/plugins/system"
> > +	_ "github.com/KPACHbIuLLIAnO4/telegraf/plugins/apache"
>
> this should be
>
> github.com/influxdb/telegraf/plugins/apache
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdb/telegraf/pull/153/files#r38257452>.
>